packages/terra-site/src/ThemeProvider.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /**
+   * The component(s) that will be wrapped by <ThemeProvider />
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Name of class for specified theme. e.g `terra-OSC-theme`
+   */
+  themeName: PropTypes.string.isRequired,
+};
+
+const ThemeProvider = ({ themeName, children, ...customProps }) => {
+  return <div className={themeName}>{children}</div>;
```

Comment:
If this is all the theme provider has, I would rather us have a prop on the Terra Base that takes in a theme, and places that on the class. This feels too small to be it's own component.